### PR TITLE
Show current player color in tuto & local

### PR DESCRIPTION
**Pull request by acieroid**

### CODE REVIEW

File: src/app/components/wrapper-components/GameWrapper.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export abstract class GameWrapper {
     @ViewChild(GameIncluderComponent)
     public gameIncluder: GameIncluderComponent;
 
-    public gameComponent: AbstractGameComponent;
+    public gameComponent: AbstractGameComponent; // TODO should be optionalized
```
Comment (MartinREMY42) [inline src/app/components/wrapper-components/GameWrapper.ts:32]: Ticket and remove TODO !

File: src/app/jscaip/tests/Coord.spec.ts [hunk 2]
```diff
@@ -55,6 +55,13 @@ describe('Coord', () => {
         expect(() => coord.getDistance(unalignedCoord))
             .toThrowError('Cannot calculate distance with non aligned coords.');
     });
+    it('should compute hexagonal alignments with isHexagonalAlignedWith', () => {
```
Comment (MartinREMY42) [inline src/app/jscaip/tests/Coord.spec.ts:58]: isHexagonallyAlignedWith !

File: src/app/components/wrapper-components/GameWrapper.ts (hunk 1 above)
Comment (acieroid) [reply src/app/components/wrapper-components/GameWrapper.ts:32]: done

File: src/app/jscaip/tests/Coord.spec.ts (hunk 2 above)
Comment (acieroid) [reply src/app/jscaip/tests/Coord.spec.ts:58]: hé oui, changé

**Review by MartinREMY42 — APPROVED** [review]

**Pull request merged**